src/components/FindMovie/FindMovie.tsx

```diff
@@ -17,41 +100,58 @@ export const FindMovie: React.FC = () => {
               id="movie-title"
               placeholder="Enter a title to search"
               className="input is-dander"
+              value={query}
+              onChange={handleChangeQuery}
             />
           </div>
 
-          <p className="help is-danger" data-cy="errorMessage">
-            Can&apos;t find a movie with such a title
-          </p>
+          {isError && (
+            <p className="help is-danger" data-cy="IsErrorMessage">
```

Comment:
```suggestion
            <p className="help is-danger" data-cy="errorMessage">
```
Don't change the original data-cy attributes, so tests will work correctly